spec/components/file_store/s3_store_spec.rb

```diff
@@ -154,6 +155,22 @@
         store.remove_upload(upload)
       end
 
+      it "removes the file from s3 on multisite", type: :multisite do
+        conn.with_connection('default') do
+          store.expects(:get_depth_for).with(upload.id).returns(0)
+          s3_helper.expects(:s3_bucket).returns(s3_bucket).at_least_once
+          upload.update_attributes!(url: "//s3-upload-bucket.s3.dualstack.us-west-1.amazonaws.com/uploads/tombstone/default/original/1X/#{upload.sha1}.png")
```

Comment:
This doesn't seem right, the URL of an upload will never include the `tombstone` path

Reply:
Ouch, fixed :+1: